arraycontext/impl/pyopencl/fake_numpy.py

```diff
@@ -265,6 +267,11 @@ def rec_equal(x, y):
             result = result.get()[()]
         return result
 
+    def astype(self, a, dtype):
```

Comment:
This is unneeded now. Should I remove it?

Reply:
if it's unneeded, I vote yes, remove! 😄